sdk/tables/azure-tables/samples/typescript/src/TableServiceClientSamples.ts

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  TableServiceClient,
+  TableClient,
+  UpdateMode,
+  EdmInt64,
+  EdmGuid,
+  odata
+} from "@azure/tables";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+// ================== Authentication ================== //
+
+const connectionString = process.env.ACCOUNT_CONNECTION_STRING || "";
+
+const accountName = process.env.ACCOUNT_NAME || "";
+const accountSas = process.env.ACCOUNT_SAS || "";
+const accountUrl = `https://${accountName}.table.core.windows.net${accountSas}`;
+
+const serviceClient = new TableServiceClient(accountUrl);
+// const serviceClient = TableServiceClient.fromConnectionString(connectionString);
+// const serviceClient = new TableServiceClient(accountUrl, new SharedKeyCredential(accountName, accountKey));
+
+const tableClient = TableClient.fromConnectionString(connectionString, "Samples");
+// const tableClient = new TableClient(accountUrl, "Samples");
+
+// ================== Table Operations ================== //
+
+async function createTable() {
+  await serviceClient.createTable("NewTable1");
+}
+
+async function deleteTable() {
+  await serviceClient.deleteTable("NewTable1");
+}
+
+async function createTableError() {
+  try {
+    await serviceClient.createTable("Samples");
+  } catch (err) {
+    console.log(err.message);
+    // {"odata.error":{"code":"TableAlreadyExists","message":{"lang":"en-US","value":"The table specified already exists."}}}
+  }
+}
+
+async function deleteTableError() {
+  try {
+    await serviceClient.deleteTable("TableDoesntExist");
+  } catch (err) {
+    console.log(err.message);
+    // {"odata.error":{"code":"ResourceNotFound","message":{"lang":"en-US","value":"The specified resource does not exist.}}}
+  }
+}
+
+async function listTables() {
+  const tables = await serviceClient.listTables({
```

Comment:
(sometimes it's interesting to "document" the defaults in the sample. 

So for instance, is there a default sort order, that kind of thing and just place the field with it's default value, indicating it's the default. I think this is just for your review, so no big deal, but something to consider.